rpm_spec/manageiq.spec.in

```diff
@@ -87,6 +87,10 @@ cd %{_builddir}
 %{__cp} -r %{gemset_builddir}/vmdb %{buildroot}%{gemset_root}
 install -m644 %{gemset_builddir}/enable %{buildroot}%{gemset_root}
 
+#copy systemd unit files
+%{__cp} %{appliance_builddir}/systemd_units/* %{buildroot}/usr/lib/systemd/system
+%{__cp} %{gemset_builddir}/bundler/gems/*/systemd_units/* %{buildroot}/usr/lib/systemd/system/
```

Comment:
Any way to limit this to just _our_ plugins?

Reply:
I'm not sure what's available to use in a spec file so I'm not really sure.  We _could_ look for engine.rb files that have a vmdb_plugin? method, like https://github.com/ManageIQ/manageiq-api/blob/e174d80c96ca3f9f796ad5aa70c680752a99f9c3/lib/manageiq/api/engine.rb#L24 .  Then only search within those directories.

Reply:
Yeah it might be a moot point as I doubt any other gem will have a `build/systemd_units` dir but figured if there was a nice way it would be worth it